returnn/config.py

```diff
@@ -190,7 +190,11 @@ def add_line(self, key, value):
       if value_type == str:
         pass  # keep as-is
       else:
-        value = eval(value)
+        try:
+          value = eval(value)
+        except:
```

Comment:
You should never catch all exceptions (including system exceptions) but always only those derived from `Exception` max, or better only those which you really expect here.

As I understood you, probably it should be:
```suggestion
        except SyntaxError:
```

Btw, PyCharm should have told you exactly about this. Please see our contribution rules.